<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingCartWindow</class>
 <widget class="QMainWindow" name="ShoppingCartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Your shopping cart</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="checkoutPushButton">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>300</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckOut</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCostLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>10</y>
      <width>571</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxshoppingWindow">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>220</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Shopping cart id of item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelPushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShopping_Cart">
    <property name="title">
     <string>Shopping Cart</string>
    </property>
   </widget>
   <addaction name="menuShopping_Cart"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
